tools/HelpGeneration/HelpGeneration.psm1

```diff
@@ -1,5 +1,5 @@
 ﻿#Requires -Modules platyPS
-function Generate-MarkdownHelp
+function New-AzMarkdownHelp
```

Comment:
The original names do not follow PowerShell standards for verbs, and generate a warning every, single build. If I had used these new names without `Az`, they conflict with the cmdlets they wrap from platyPS (which has cmdlets with the same name). Since these functions aren't published, but are used to generate Az documentation, I thought Az to be an appropriate prefix to not cause issues with platyPS.